### PR TITLE
Add uninstall script for KDE store

### PR DESCRIPTION
**Pull request by alex1701c**
Hello,

this script is needed for the upcoming KDE store integration, as mentioned in https://github.com/domschrei/krunner-symbols/issues/17.

I would also ask you to consider creating binary packages, this is extremely useful for inexperienced users and you don't have to install the bunch of build packages.
This can be achieved relatively easily using cpack, if you would like help for this just say it :-).

### CODE REVIEW

File: uninstall.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Exit immediately if something fails
+set -e
+
+cd build
+sudo make uninstall
+kquitapp5 krunner 2> /dev/null; kstart5 --windowclass krunner krunner > /dev/null 2>&1 &
```
Comment (SuperSandro2000) [inline uninstall.sh]:
```suggestion
kquitapp5 krunner 2> /dev/null; kstart5 --windowclass krunner krunner &> /dev/null &
```

Comment (alex1701c) [reply uninstall.sh]: Actually we can simplify this without the pipes now, some time ago we got annoying warnings.

**Comment by domschrei** [comment]
Thanks a lot!

Preparing a binary sounds like a really good idea :) I am currently on vacation, but when I come back (~2 weeks) I'll take a look at CPack. I'll also be happy to help to (finally!) get the plugin into the KDE Store.

**Comment by alex1701c** [comment]
Then enjoy your vacation :-)

**Pull request merged**